src/workers/cdc/common.ts

```diff
@@ -8,3 +19,76 @@ export const getTableName = <Entity>(
   con: DataSource,
   target: EntityTarget<Entity>,
 ): string => con.getRepository(target).metadata.tableName;
+
+export const notifyPostContentUpdated = async ({
+  con,
+  logger,
+  post,
+}: {
+  con: DataSource;
+  logger: FastifyBaseLogger;
+  post: ChangeObject<Post>;
+}): Promise<void> => {
+  const [source, keywords, relatedPosts] = await Promise.all([
+    con.getRepository(Source).findOneBy({
+      id: post.sourceId,
+    }),
+    con.getRepository(PostKeyword).findBy({
+      postId: post.id,
+    }),
+    con.getRepository(PostRelation).findBy({
+      postId: post.id,
+    }),
+  ]);
+  const articlePost = post as ChangeObject<ArticlePost>;
+  const freeformPost = post as ChangeObject<FreeformPost>;
+  const machineSource = source as MachineSource;
+
+  const contentUpdatedMessage = new ContentUpdatedMessage({
+    yggdrasilId: post.yggdrasilId,
+    postId: post.id,
+    type: post.type,
+    title: post.title,
+    createdAt: post.createdAt,
+    updatedAt: post.metadataChangedAt,
+    source: {
+      ...source,
+      createdAt: +source.createdAt,
+      // TODO issue with null can't be parsed as JSON
+      headerImage: source.headerImage || undefined,
+      color: source.color || undefined,
+      twitter: machineSource.twitter || undefined,
+      website: machineSource.website || undefined,
+      description: source.description || undefined,
```

Comment:
https://github.com/bufbuild/protobuf-es/issues/860, I might have to fallback `null` values, I am not even sure why this happens

Reply:
Looks like they are gonna port the fix so our issue will be fixed.

Reply:
I also adjusted schema so it supports it when they fix it on their end https://github.com/dailydotdev/schema/pull/27

Reply:
Fixed in 55f7487fde1a9e6b4695cc8a98e10f31a54351ed